ERROR: Feature "SSH" (Unknown) failed to install

Autogenerated scenario: ssh against mcr.microsoft.com/devcontainers/base:alpine

```
#15 [dev_containers_target_stage 5/5] RUN --mount=type=bind,from=dev_containers_feature_content_source,source=ssh_0,target=/tmp/build-features-src/ssh_0     cp -ar /tmp/build-features-src/ssh_0 /tmp/dev-container-features  && chmod -R 0755 /tmp/dev-container-features/ssh_0  && cd /tmp/dev-container-features/ssh_0  && chmod +x ./devcontainer-features-install.sh  && ./devcontainer-features-install.sh  && rm -rf /tmp/dev-container-features/ssh_0

[2025-02-26T06:27:13.726Z] #15 0.151 ===========================================================================
#15 0.151 Feature       : SSH
#15 0.151 Description   : Sets up SSH into the Dev Environment
#15 0.151 Id            : Unknown
[2025-02-26T06:27:13.726Z] 
#15 0.151 Version       : 1.2.0
#15 0.151 Documentation : https://github.com/Automattic/vip-codespaces/tree/trunk/features/src/ssh
#15 0.151 Options       :
#15 0.151     ENABLED="true"
#15 0.151     PORT="22"
#15 0.151     NEW_PASSWORD="skip"
#15 0.151 ===========================================================================

[2025-02-26T06:27:13.840Z] #15 0.155 (*) Installing OpenSSH server...
#15 0.165 fetch https://dl-cdn.alpinelinux.org/alpine/v3.20/main/x86_64/APKINDEX.tar.gz
#15 0.265 fetch https://dl-cdn.alpinelinux.org/alpine/v3.20/community/x86_64/APKINDEX.tar.gz

[2025-02-26T06:27:14.123Z] #15 0.549 ERROR: unable to select packages:

[2025-02-26T06:27:14.217Z] #15 0.551   openssh-client-common-9.7_p1-r4:
#15 0.551     breaks: openssh-client-default-9.7_p1-r5[openssh-client-common=9.7_p1-r5]
#15 0.609 ERROR: Feature "SSH" (Unknown) failed to install! Look at the documentation at https://github.com/Automattic/vip-codespaces/tree/trunk/features/src/ssh for help troubleshooting this error.
```

Ref: https://github.com/Automattic/vip-codespaces/actions/runs/13533454411/job/37832727758

```diff
diff --git a/features/src/ssh/install.sh b/features/src/ssh/install.sh
index b11a65f..a2066dc 100755
--- a/features/src/ssh/install.sh
+++ b/features/src/ssh/install.sh
@@ -36,6 +36,7 @@ if [ "${ENABLED:-}" = "true" ]; then
         ;;
 
         "alpine")
+            apk upgrade --no-cache 'openssh*' openssl
             # shellcheck disable=SC2086
             apk add --no-cache openssh-server-pam openssl
             ln -sf /usr/sbin/sshd.pam /usr/sbin/sshd
```

--- a/features/src/ssh/devcontainer-feature.json
+++ b/features/src/ssh/devcontainer-feature.json
@@ -2,7 +2,7 @@
     "id": "ssh",
     "name": "SSH",
     "description": "Sets up SSH into the Dev Environment",
-    "version": "1.2.0",
+    "version": "1.2.1",
     "documentationURL": "https://github.com/Automattic/vip-codespaces/tree/trunk/features/src/ssh",
     "options": {
         "enabled": {

--- a/features/src/ssh/install.sh
+++ b/features/src/ssh/install.sh
@@ -36,6 +36,7 @@ if [ "${ENABLED:-}" = "true" ]; then
         ;;
 
         "alpine")
+            apk upgrade --no-cache 'openssh*' openssl
             # shellcheck disable=SC2086
             apk add --no-cache openssh-server-pam openssl
             ln -sf /usr/sbin/sshd.pam /usr/sbin/sshd

